in_toto/runlib.py

```diff
@@ -279,7 +279,7 @@ def _duplicate_streams():
     return proc.poll(), streams["out"], streams["err"]
 
 
-def execute_link(link_cmd_args, record_streams):
+def execute_link(link_cmd_args, record_streams, timeout=10):
```

Comment:
We can drop the default value here as the expectation is this is private. AFAICT it's not documented on https://in-toto.readthedocs.io/en/latest/search.html?q=execute_link&check_keywords=yes&area=default